src/validate.js

```diff
@@ -0,0 +1,34 @@
+'use-strict';
```

Comment:
Is this compiled from the Typescript? If so, we should `.gitignore` this file and `git rm --cached src/validate.js` to remove it from the pull request.

Reply:
Yep, was automatically compiled from typescript to a JS file, I'll remove it!